src/components/CardListItem.js

```diff
@@ -14,7 +14,7 @@ export class CardListItem extends React.PureComponent {
     const { navigation, horizontal, item } = this.props;
     const { routeName, params, image, category, name } = item;
 
-    // TODO accessibility logic needs to be implemented
+    // TODO count articles logic could to be implemented
```

Comment:
this logic to count could also be applied in the TextList, so that the user can know the index of the element she/he clicked on

Reply:
good idea, we will put it on a roadmap for the future